packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -285,7 +285,12 @@ export type PluginExtensionDataSourceConfigContext<
   setSecureJsonData: (secureJsonData: SecureJsonData) => void;
 };
 
-export type PluginExtensionCommandPaletteContext = {};
+export type PluginExtensionCommandPaletteContext = {
```

Comment:
Can we instead create a separate type? I assume this needs to be optional because we reuse it also for static results, but I assume we can just create `DynamicPluginExtensionCommandPaletteContext` or something like that, that would have this non optional which I think makese more sense.